pkg/oauth/util/discovery.go

```diff
@@ -51,14 +50,13 @@ type OauthAuthorizationServerMetadata struct {
 // validate configuration using LoadOAuthMetadataFile
 
 func getOauthMetadata(masterPublicURL string) OauthAuthorizationServerMetadata {
-	config := osinserver.NewDefaultServerConfig()
```

Comment:
Will the import restriction stuff prevent someone from adding this back in the future?  If not probably best just to comment it out instead of deleting all together.

Reply:
> Will the import restriction stuff prevent someone from adding this back in the future? If not probably best just to comment it out instead of deleting all together.

Yes, it will